lib/venue.rb

```diff
@@ -1,2 +1,37 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+
+  def add_patron(name)
+    @patrons.push(name)
+  end
+
+  def yell_at_patrons
+    yelling = []
+    patrons.each do |yell|
+      yelling << yell.upcase
+    end
+    yelling
```

Comment:
Looks good! We haven't talked about it in class yet but this would be a good place for an enumerable as well.